src/modules/shlvl.rs

```diff
@@ -20,10 +22,21 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 
     let shlvl_str = &shlvl.to_string();
 
+    let repeat_count = if config.repeat {
+        shlvl.try_into().unwrap_or(1)
+    } else {
+        1
+    };
+    let symbol = if repeat_count != 1 {
+        Cow::Owned(config.symbol.repeat(repeat_count))
+    } else {
+        Cow::Borrowed(config.symbol)
+    };
```

Comment:
I might be missing something but could this not be
```suggestion
    let symbol = config.symbol.repeat(repeat_count);
```

Reply:
I figured it would be a slight optimization to not have to allocate and copy the symbol string in the common case where `repeat_count` is 1 by using `Cow::Borrowed`.